etc/mtree/BSD.debug.dist

```diff
@@ -30,7 +30,7 @@
                 c18n
                 ..
                 clang
-                    18
+                    17
```

Comment:
I don't think this is correct.  We switched to building the in-tree sanitizers a while ago which is what this is for and this follows the upstream LLVM version, not CHERI LLVM.